src/migrations/00082-pinejs-15.sql

```diff
@@ -0,0 +1,36 @@
+-- Boolean type conversions
+
+ALTER TABLE "application type"
+ALTER COLUMN "supports web url" SET DATA TYPE BOOLEAN USING b::BOOLEAN,
```

Comment:
```suggestion
ALTER COLUMN "supports web url" SET DATA TYPE BOOLEAN USING "supports web url"::BOOLEAN,
```
Applies in all conversions.